.github/workflows/check-examples.yml

```diff
@@ -0,0 +1,60 @@
+name: Check example notebooks
+
+on:
+  pull_request:
+    branches:
+      - main
+      - develop
+  schedule:
+    # 04:00 every Saturday morning
+    - cron: '0 4 * * 6'
+
+env:
+  PYTKET_REMOTE_QUANTINUUM_USERNAME: ${{ secrets.PYTKET_REMOTE_QUANTINUUM_USERNAME }}
+  PYTKET_REMOTE_QUANTINUUM_PASSWORD: ${{ secrets.PYTKET_REMOTE_QUANTINUUM_PASSWORD }}
+
+jobs:
+
+  changes:
+    runs-on: ubuntu-20.04
+    outputs:
+      examples: ${{ steps.filter.outputs.examples }}
+    steps:
+    - uses: actions/checkout@v3
+    - uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        base: ${{ github.ref }}
+        filters: |
+          examples:
+            - 'examples/**'
+            - '.github/workflows/check-examples.yml'
+
+  check:
+    name: Check notebooks
+    needs: changes
+    if: github.event_name == 'schedule' || needs.changes.outputs.examples == 'true'
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10']
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: '0'
+    - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/* +refs/heads/*:refs/remotes/origin/*
```

Comment:
This is not actually needed, and wastes a bit of time.
```suggestion
```

Reply:
Will remove :D